### PR TITLE
add antiscroll support

### PR DESCRIPTION
**Pull request by bcherny**

### CODE REVIEW

File: slick.grid.css.map [hunk 1]
```diff
@@ -1,7 +1,7 @@
...
...,EAAE,QAAQ;EAClB,WAAW,EAAE,MAAM;EACnB,MAAM,EAAE,OAAO;EACf,QAAQ,EAAE,MAAM;EAChB,aAAa,EAAE,gBAAgB;EAC/B,UAAU,EAAE,OAAO;;AAInB;2BAAK;EACH,YAAY,EAAE,gBAAgB;EAC9B,WAAW,EAAE,CAAC;EACd,UAAU,EAAE,CAAC;EACb,aAAa,EAAE,CAAC;EAEhB;oCAAQ;IACN,UAAU,EAAE,WAAW;;AAE7B,qBAAqB;EACnB,UAAU,EAAE,IAAI;EAChB,iCAAW;IACT,UAAU,EAAE,UAAU;IACtB,KAAK,EAAE,IAAI;IACX,MAAM,EAAE,IAAI;;AAKhB,2BAA2B;EACzB,gBAAgB,EAAE,OAAO;;AAC3B,kCAAkC;EAChC,gBAAgB,EAAE,OAAO;;AAI3B,qBAAqB;EACnB,OAAO,EAAE,YAAY;EACrB,KAAK,EAAE,GAAG;EACV,MAAM,EAAE,GAAG;EACX,WAAW,EAAE,GAAG;EAChB,UAAU,EAAE,GAAG;EACf,KAAK,EAAE,IAAI;;AAEb,0BAA0B;EACxB,UAAU,EAAE,yBAAyB;;AAEvC,yBAAyB;EACvB,UAAU,EAAE,wBAAwB;;AAEtC,mBAAmB;EACjB,QAAQ,EAAE,QAAQ;EAClB,SAAS,EAAE,KAAK;EAChB,OAAO,EAAE,KAAK;EACd,MAAM,EAAE,UAAU;EAClB,KAAK,EAAE,GAAG;EACV,KAAK,EAAE,GAAG;EACV,GAAG,EAAE,CAAC;EACN,MAAM,EAAE,IAAI;;AAEd,2BAA2B;EACzB,UAAU,EAAE,MAAM;;AAMpB,mBAAmB;EACjB,OAAO,EAAE,YAAY;;AAIvB,oBAAoB;EAClB,OAAO,EAAE,YAAY;EACrB,UAAU,EAAE,IAAI;EAChB,OAAO,EAAE,IAAI;EACb,MAAM,EAAE,iBAAmB;EAC3B,MAAM,EAAE,IAAI;;AAEd,oBAAoB;EAClB,OAAO,EAAE,YAAY;EACrB,MAAM,EAAE,GAAG;EACX,UAAU,EAAE,IAAI;EAChB,OAAO,EAAE,GAAG;EACZ,MAAM,EAAE,iBAAmB;;AAE7B,gBAAgB;EACd,OAAO,EAAE,EAAE;EACX,QAAQ,EAAE,QAAQ;EAClB,MAAM,EAAE,gBAAgB",
```
Comment (albertywu) [inline slick.grid.css.map]: why is this being checked in?

Comment (bcherny) [inline slick.grid.css.map]: i'm not sure - it was already being checked in. i think it would be reasonable to remove it.

File: slick.grid.js [hunk 2]
```diff
@@ -1114,6 +1139,46 @@ if (typeof Slick === "undefined") {
...
+      }
+
+    }
+
+    function enableAntiscroll ($element) {
+
+      $element
+        .addClass('antiscroll-wrap')
+        .antiscroll({
+          autoShow: false
```
Comment (albertywu) [inline slick.grid.js]: might be nice to have this as a config option?

File: slick.grid.js [hunk 3]
```diff
@@ -2046,7 +2113,13 @@ if (typeof Slick === "undefined") {
...
 
       var topOffset = topViewport.el.height(); // the top boundary of the center row of things
-      contentViewport.el.css({ 'top': topOffset, 'height': contentViewport.height });
+      contentViewportWrap.el.css({ 'top': topOffset, 'height': contentViewport.height });
+
+      // something is setting the contentViewport's height, and should't be.
+      // this causes the viewport to not resize when the window is resized.
+      // as a workaround, override the CSS here.
+      // TODO: figure out what's setting the height and fix it there instead.
+      contentViewport.el.css({ top: 0, height: '100%', width: '100%' });
```
Comment (albertywu) [inline slick.grid.js]: what was the contentViewport's height being set to?

File: slick.grid.js (hunk 2 above)
Comment (bcherny) [inline slick.grid.js]: added

File: slick.grid.js (hunk 3 above)
Comment (bcherny) [inline slick.grid.js]: the same as `contentViewportWrap`

File: slick.grid.css.map (hunk 1 above)
Comment (albertywu) [inline slick.grid.css.map]: i'd prefer to not check in .css, .map but I don't feel right taking them out

**Comment by albertywu** [comment]
lgtm

File: slick.grid.css.map (hunk 1 above)
Comment (bcherny) [inline slick.grid.css.map]: agreed. css might make sense to keep since it's the preferred consumable. map could be helpful for debugging.

**Pull request merged**